parts/k8s/cloud-init/artifacts/docker-monitor.timer

```diff
@@ -1,7 +1,7 @@
 [Unit]
```

Comment:
I would advocate we eliminate this timer spec. It adds complexity to the overall implementation, and the way we're implementing the docker health check (docker ps) should not be "racy" given that the health check script runs *only after* the docker systemd service has started (see `After=docker.service` above)

@Michael-Sinz @mboersma  thoughts?

Reply:
I think what we're implicitly saying by delaying things for 30 mins is that we are tolerant of `docker ps` repeatedly failing during the first 30 mins of boot time, which I don't think is defensible.

Reply:
I think providing a warmup period for Docker to get its act together was basically what Azure/acs-engine#4050 was about. I agree simpler is way better when it comes to systemd units especially, so l'm ok with removing it to see if it's unneeded now.

Reply:
After further investigation, it *is* a bit tricky to tell a systemd service "wait until this service starts, and *also* until it is fully activated; and so I've moved the delay into the health script itself. Arguably that is less complicated than maintaining systemd overhead to do that.